dags/chronos_train_dag.py

```
@@ -0,0 +1,344 @@
+        train_data = TimeSeriesDataFrame.from_data_frame(train_data_df)
+
+        # Load the AutoGluon TimeSeriesPredictor from the given path
+        predictor = TimeSeriesPredictor.load(model_info['model_path'])
+
+        predictions = predictor.predict(train_data)
+        logging.info("Finished predict: Predictions generated")
+
+        # Log predictions to the same MLflow run
+        mlflow.set_tracking_uri(MLFLOW_TRACKING_URI)
+        with mlflow.start_run(run_id=model_info['run_id']):
+            mlflow.log_artifact('/tmp/predictions.csv', 'predictions')
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp/predictions.csv"

(S108)